go.mod

```diff
@@ -3,20 +3,44 @@ module github.com/litmuschaos/litmusctl
 go 1.16
 
 require (
-	github.com/Azure/go-autorest/autorest v0.11.18 // indirect
+	github.com/argoproj/argo-workflows/v3 v3.2.3
 	github.com/fatih/color v1.12.0
 	github.com/golang-jwt/jwt v3.2.1+incompatible
-	github.com/imdario/mergo v0.3.11 // indirect
+	github.com/litmuschaos/chaos-operator v0.0.0-20211109081056-46bcd99c3daf
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/viper v1.8.1
-	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad // indirect
-	golang.org/x/sys v0.0.0-20210616094352-59db8d763f22 // indirect
-	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
-	golang.org/x/text v0.3.6 // indirect
+	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.20.4
-	k8s.io/apimachinery v0.20.4
-	k8s.io/client-go v0.20.4
-	k8s.io/utils v0.0.0-20210111153108-fddb29f9d009 // indirect
+	k8s.io/api v0.22.1
+	k8s.io/apimachinery v0.22.1
+	k8s.io/client-go v12.0.0+incompatible
+	sigs.k8s.io/yaml v1.2.0
```

Comment:
I've imported this yaml library instead of using the gopkg one.

This is done because this yaml library can convert to JSON before unmarshaling. This means that while unmarshaling the workflow yaml I can directly import the workflow struct from argo-workflows.

If not for this, we'll need to add the whole type struct with `yaml:"<name>"` annotations to each of the struct fields.